modules/cluster-management/src/testFixtures/java/org/apache/ignite/internal/cluster/management/raft/TestClusterStateStorage.java

```diff
@@ -50,6 +55,8 @@
 public class TestClusterStateStorage implements ClusterStateStorage {
     private static final String SNAPSHOT_FILE = "snapshot.bin";
 
+    private static final CmgMessagesFactory cmgMessagesFactory = new CmgMessagesFactory();
```

Comment:
Since the field is static, maybe it would be more correct to name it **CMG_MESSAGES_FACTORY**?